libmamba/src/api/configuration.cpp

```diff
@@ -718,6 +781,17 @@ namespace mamba
                     validate_existing_root_prefix(default_root_prefix_v1())
                         .or_else([](const auto& /* error */)
                                  { return validate_root_prefix(default_root_prefix_v2()); })
```

Comment:
This is actually making sure that the root prefix will be set to `fs::u8path(util::user_data_dir()) / "mamba"` (which is in most cases something like `/home/user/.local/share/mamba`)  if it doesn't already exist.
It it exists, it checks that it's a directory containing at least `pkgs`, `conda-meta` or `envs` directories.
So mamba will only complain if that directory exists and doesn't contain one of the mentioned dirs (which I don't think happens a lot...).
In this case, we set the root prefix to the mamba installation root prefix.